### PR TITLE
replace alias_method_chain by Module#prepend

### PR DESCRIPTION
**Pull request by mathieujobin**
this eliminates the annoying deprecation warning on rails 5
and everything works fine 

it requires ruby 2.1 though I would just bump the requirement to be 2.2.2 like rails 5

### CODE REVIEW

File: lib/pickle/session/parser.rb [hunk 1]
```diff
@@ -2,8 +2,18 @@ module Pickle
...
+            {$1 => session.model!($2)}
+          else
+            super(field)
+          end
+        end
+      end
+
       def self.included(parser_class)
-        parser_class.alias_method_chain :parse_field, :model
+        parser_class.prepend ParseFieldWithModel
```
Comment (mathieujobin) [inline lib/pickle/session/parser.rb]: public `prepend` requires ruby 2.1, but if I do `.send(:prepend, ...)` then ruby 2.0 is fine

**Comment by mathieujobin** [comment]
addresses #69

**Comment by mathieujobin** [comment]
any chance this could get merged and #69 closed? 

travis errors looks unrelated

**Comment by rosskevin** [comment]
Thanks @mathieujobin - works for me.

Last release was January 2015.  Is this repo abandoned?

**Comment by mathieujobin** [comment]
does pickle still has a maintainer? could we get this merged in and release?

I am thinking we should create a `pickle` team, move this to `https://github.com/pickle/pickle`
have a list of maintainers, etc.

**Comment by rosskevin** [comment]
@mathieujobin, this is mit licensed. If we get no response, and considering the inactivity, I would support forking and trying​ to foster a community of maintainers. We still rely on it so I don't want to see it fall behind.

**Comment by ianwhite** [comment]
Hello all,

I'm the creator of pickle.  Currently @niklas and @chewi are maintainers.
I fully support creating pickle/pickle and creating a pickle team.

In the meantime, to ease this transition, would @mathieujobin and @rosskevin like to be admins for this repository?

Cheers,
Ian

**Comment by mathieujobin** [comment]
Sure

Can we also merge this and tag 0.5.2 or 0.6.0?

Which ever you prefer @ianwhite

**Comment by ianwhite** [comment]
I don't use pickle in any of my current work, since about 4 years ago. @niklas and @chewi picked up the maintenance. Do you guys still use pickle, and want to continue to maintain it?

@mathieujobin I've added you as a collaborator to this repo.  If you want rubygem collaborator status, send your email address to me on ian.w.white@gmail.com and I'll add you as an owner.

**Comment by mathieujobin** [comment]
this was tagged as v0.5.2
https://github.com/ianwhite/pickle/compare/v0.5.1...v0.5.2

will push to rubygems when I get access

Thanks All

**Comment by mathieujobin** [comment]
v0.5.2 pushed to rubygems

**Pull request merged**